dhis2/src/main/java/org/motechproject/dhis2/rest/service/impl/DhisWebServiceImpl.java

```diff
@@ -420,6 +421,14 @@ private DhisStatusResponse createEntity(Settings settings, String uri, String js
             closeResponse(response);
         }
 
+        LOGGER.debug(String.format("DHIS2 status response: %s", status.getStatus().toString()));
+
+        if (status.getStatus() == DhisStatus.ERROR) {
```

Comment:
Did you check it correctly ? In the ticket description on JIRA there is an information that response can return 200 (OK) but response body may contain 'ERROR', so it seems that you do not include this condition